hardcoded.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+// Used to generate data for consts.php INSTALLER_TO_REPO_MINOR_VERSIONS
+// Copy .cow.pat.json from the current release into the directory this script is being run from (file is .gitignored)
+
+include 'consts.php';
+
+$filename = '.cow.pat.json';
+if (!file_exists($filename)) {
+    throw new \RuntimeException("Copy $filename into the directory this is being called from");
+}
+
+$versions = [];
+
+function repoName(string $name)
+{
+    $prefix = '';
+    if (!preg_match('#^(cwp/)#', $name) &&
+        !preg_match('#(/recipe-|/silverstripe-|/comment-notifications$)#', $name)
+    ) {
+        $prefix = 'silverstripe-';
+    }
+    if (preg_match('#(/agency-extensions|/starter-theme|/watea-theme)$#', $name)) {
+        $prefix = 'cwp-';
+    }
+    $name = str_replace('/cwp-agency-extensions', '/cwp-agencyextensions', $name);
```

Comment:
Prefix isn't applied yet - it's still `agency-extensions`. I still think it's easier to just put this inside the above condition but if we're leaving it here we need to `preg_replace` and add `$` to the end of the regex to protect against `agency-extensions-something`

Reply:
updated

gridfieldextensins missing because it's not in .cow.pat.json.  I've manually added it to the bottom of the list for 4.10 and 4.11